server.js

```diff
@@ -1,6 +1,6 @@
 var express = require('express');
 var logger = require('morgan');
-
+// changes
```

Comment:
Looks good for now. This is what I would do moving forward.

- [x] A
- [x] B
- [x] C